pkg/diskmaker/controllers/lvset/add.go

```diff
@@ -0,0 +1,55 @@
+package lvset
+
+import (
+	localv1alpha1 "github.com/openshift/local-storage-operator/pkg/apis/local/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+var log = logf.Log.WithName("controller_localvolumeset")
+
+// Add adds a new nodeside lvset controller to mgr
+func Add(mgr manager.Manager) error {
+	r := &ReconcileLocalVolumeSet{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	// Create a new controller
+	c, err := controller.New("localvolumeset-controller", mgr, controller.Options{Reconciler: r})
```

Comment:
same name for both controller on the node and controller on the control-plane? in fact, I would like us to move away from name "controller" for components that are running on the node

Reply:
`controller` here is the name of the `controller-runtime` struct.

Reply:
I meant `localvolumeset-controller` string

Reply:
Also, what I meant by moving away from controller for components running on node is - if you open `cmd/diskmaker` folder, you will notice two files - "diskmaker.go" and "controller.go". What is "controller.go" supposed to mean in that context? How is it different from "diskmaker.go"?

Reply:
fixed